grakn-core/src/main/java/ai/grakn/engine/TaskId.java

```diff
@@ -19,16 +19,18 @@
 
 package ai.grakn.engine;
 
-import javax.annotation.CheckReturnValue;
 import java.util.UUID;
+import javax.annotation.CheckReturnValue;
+import org.apache.tinkerpop.shaded.jackson.annotation.JsonCreator;
+import org.apache.tinkerpop.shaded.jackson.annotation.JsonProperty;
 
 /**
  * An identifier for a task
  *
  * @author Felix Chapman
  */
 public final class TaskId {
-    private final String value;
```

Comment:
Jackson serialisation doesn't work with final (we can convert this to Autovalue+Builder in the future)